### PR TITLE
end-to-end example of a worfklow of MLflow model deploying with seldon-core

### PR DESCRIPTION
**Pull request by jsuchome**
Requires https://github.com/fuseml/extensions/pull/31

### CODE REVIEW

File: prediction/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+NOTE: Use full json related headers to curl request for seldon-core based prediction service. (See https://github.com/SeldonIO/seldon-core/issues/1770)
+
+For example
+
+```bash
+curl -d @prediction/data-sklearn-seldon.json -X POST http://$INGRESS_HOST/seldon/fuseml-workloads/project-02-mlflow-seldon/api/v1.0/predictions -H "Accept: application/json" -H "Content-Type: application/json"
+```
+
+(You'll get the rught URL from `fuseml application list` call)
```
Comment (flaviodsr) [inline prediction/README.md:11]: typo: `rught`

**Review by flaviodsr — CHANGES_REQUESTED** [review]
The predictor step on the kfserving workflow "makes use" of the kfserving extension. Although I think this is not really needed/implemented, I wonder if it makes sense to also have the predictor step on this new workflow make use of the seldon-core extension just to keep it in sync.

File: prediction/README.md (hunk 1 above)
Comment (jsuchome) [reply prediction/README.md]: fixed

**Review by flaviodsr — APPROVED** [review]

File: workflows/mlflow-seldon-e2e.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      - name: predictor
+        value: '{{ inputs.predictor }}'
+      - name: mlflow-codeset
+        codeset:
+          name: '{{ inputs.mlflow-codeset }}'
+          path: '/project'
+    outputs:
+      - name: prediction-url
+    extensions:
+      - name: s3-storage
```
Comment (stefannica) [inline workflows/mlflow-seldon-e2e.yaml:58]: Have to agree with Flavio, you should reference the seldon core service here, same as we do with KFServing

**Review by stefannica — COMMENTED** [review]
The workflow should reference the extensions that it uses. Otherwise looks good to me.

File: workflows/mlflow-seldon-e2e.yaml (hunk 2 above)
Comment (jsuchome) [reply workflows/mlflow-seldon-e2e.yaml:58]: Hm, I think there might actually not be any service to expose when I install only "core-operator" ....

Comment (stefannica) [reply workflows/mlflow-seldon-e2e.yaml:58]: true, but the workflow doesn't reference extension services that are exposed here. think about it like this: even if you didn't expose the S3 service through an istio virtualservice, you would still reference it here and access it through the internal endpoint. same goes with kfserving. and with seldon core

**Pull request merged**